content/pages/sprint/2021/08end-devenv.rst

```diff
@@ -0,0 +1,75 @@
+============================================
+Scisprint 2021 End of August: solvcon devenv
+============================================
+
+:date: 2021-07-25 14:15
+:url: sprint/2021/08end-devenv
+:save_as: sprint/2021/08end-devenv.html
+
+Scisprint 2021 End of August: solvcon devenv
+============================================
+
+We cordially invite you to join the short sprint for the development
+environment for `solvcon <https://solvcon.net/>`_ projects.  solvcon_ is a host
+of projects focusing on numerically solving the governing equations of
+transient phenomena.
+
+The development environment, i.e., devenv (see `the github page
+<https://github.com/solvcon/devenv>`__), is also a solvcon project, but has a
+much broader scope that includes the infrastructure for developing code for
+numerical calculations, which are the foundation of computational sciences and
+mathematics.  The real-world applications, including physics, mechanics,
+semiconductor, finance, machine learning, etc., will be built with it.
+
+Numerical calculations require writing high-performance code.  The complex
+implementations call for tremendous amount of code to be written and it is not
+possible to do it all by a single team.  External dependencies are necessary.
+However, because of the requirements of accuracy and performance, the
+dependencies are usually at the source level.  Building dependencies from
+source is the norm.  The project devenv is made to help the source-level
+dependency management and alignment for the runtime among developers.
+
+Registration
+------------
+
+Please join `gather.town <https://gather.town/app/yLTe8mBDb8pogMOX/sciwork>`_
```

Comment:
If we do multple projects, perhaps this sprint should use a kktix for registration.

Reply:
Ok. Let me study how to use kktix for registration.
Does Sciwork have official kktix account?

Reply:
Yes we do: https://sciwork.kktix.cc .  What's your kktix account?

Reply:
kktix account: j8xixo12

Reply:
@j8xixo12 Can you provide me the email you use for the kktix account.  kktix seems to only allow me to invite using email.  Sorry about that.

Reply:
okok, email: `as2266317@gmail.com`

Reply:
Thanks.  Invitation sent.